ArduinoUno.md

```diff
@@ -0,0 +1,228 @@
+## Arduino Uno 
+
+The Arduino Uno connector is a standardised connector in Mbed, which has a set amount of exposed functionality. To achieve meaningful portability of application code across various Mbed enabled boards that are Arduino Uno compliant, the pin names used for the connector pins are common across these boards.
```

Comment:
Please rename "Mbed enabled" --> "Mbed".
We'll handle Mbed Enabled separately as specific requirements :)

Reply:
Yeah I will correct that